website/mails/listeners.py

```diff
@@ -32,9 +31,8 @@ def queue_first_public_project_email(user, node, meeting_creation):
     """
     from osf.models.queued_mail import queue_mail, QueuedMail, NEW_PUBLIC_PROJECT_TYPE, NEW_PUBLIC_PROJECT
     if not meeting_creation:
-        sent_mail = QueuedMail.find(Q('user', 'eq', user) & Q('sent_at', 'ne', None) &
-                                          Q('email_type', 'eq', NEW_PUBLIC_PROJECT_TYPE))
-        if not sent_mail.count():
+        sent_mail = QueuedMail.objects.filter(user=user, sent_at__isnull=False, email_type=NEW_PUBLIC_PROJECT_TYPE)
```

Comment:
in the `QueuedMail` class the `send_at` field has `null=False`, so you might not need that middle query filter param